service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/SpecimenMapperTest.java

```diff
@@ -111,7 +111,7 @@ public void When_MappingDefaultSpecimenJson_Expect_DefaultSpecimenStatementXmlOu
             TEST_FILE_DIRECTORY + "expected_output_default_specimen.xml"
         );
 
-        Observation observation = new Observation().setSpecimen(new Reference().setReference("Specimen/Default-1"));
+        Observation observation = new Observation().setSpecimen(new Reference().setReference("Specimen/DUMMY-SPECIMEN"));
```

Comment:
I think, It could be empty observation here, without specimen reference, because in case the specimen is default, all observations not related to dummy specimen are used. So the reference should not be used in this case.

Reply:
This test is testing for a default specimen (i.e. no specimens) and a default observation (i.e. no observations).
I've added another test (`When_MappingDefaultSpecimenWithObservations_Expect_DefaultSpecimenAndObservationsXmlOutput`) for a default specimen (i.e. no specimens) but there is observations)